code/modules/organs/external/_external.dm

```diff
@@ -272,39 +265,52 @@
 			E.update_icon()
 			return
 
+	//Allow removing sub-limbs
+	if(istype(W,/obj/item/circular_saw) && LAZYLEN(children))
+		var/list/removables = get_limbs_recursive(TRUE)
+		if(LAZYLEN(removables))
+			var/obj/item/organ/external/removing = pick(removables)
+			if(do_after(user, 3 SECONDS, removing, FALSE))
```

Comment:
Probably should check if the organ is still attached and present in the mob.

Reply:
That's not something you can do on a mob. Its for loose limbs lying on the ground. Added a check if its in the parent limb instead.